modules/packages/UnorderedCopy.chpl

```diff
@@ -76,33 +82,32 @@
  */
 module UnorderedCopy {
   /*
-     Unordered copy. Only supported for identical numeric and bool types.
+     Unordered copy. Only supported for identical trivially copyable types.
```

Comment:
I don't get the part about `identical`. AFAIK you are not checking that anywhere?

Reply:
That makes me wonder if the "last resort" error message should leave out the bit about identical types? Since if the problem was that the types differ (but they were both POD), then they would not get that error?

Reply:
I like keeping the error message consistent.

Reply:
OK, works for me.